lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -1,45 +1,91 @@
                     </button>
                 </div>
+            </adf-content-metadata-header>
+        </mat-expansion-panel-header>
+        <adf-card-view
+            (keydown)="keyDown($event)"
+            [properties]="basicProperties$ | async"
+            [editable]="isEditingGeneralInfo"
+            [displayEmpty]="displayEmpty"
+            [copyToClipboardAction]="copyToClipboardAction"
+            [useChipsForMultiValueProperty]="useChipsForMultiValueProperty"
+            [multiValueSeparator]="multiValueSeparator">
+        </adf-card-view>
+    </mat-expansion-panel>
+
+    <ng-container *ngIf="displayTags">
+        <mat-expansion-panel hideToggle [(expanded)]="isTagPanelExpanded">
+            <mat-expansion-panel-header>
+                <adf-content-metadata-header [title]="'METADATA.BASIC.TAGS'" [expanded]="isTagPanelExpanded">
+                    <div class="adf-tags-buttons">
+                        <button *ngIf="canEditTags"
+                                mat-icon-button
+                                (click)="onToggleTagsEdit($event)"
+                                [attr.title]="'CORE.METADATA.ACTIONS.EDIT' | translate"
+                                [attr.aria-label]="'CORE.METADATA.ACCESSIBILITY.EDIT' | translate"
+                                data-automation-id="showing-tag-input-button"
+                                class="adf-edit-icon-buttons">
+                            <mat-icon>mode_edit</mat-icon>
+                        </button>
+                    </div>
+                    <div *ngIf="isEditingTags" class="adf-metadata-action-buttons">
+                        <button mat-icon-button
+                                [attr.title]="'CORE.METADATA.ACTIONS.CANCEL' | translate"
+                                (click)="onCancelTagsEdit($event)"
+                                data-automation-id="reset-tags-metadata"
+                                class="adf-metadata-action-buttons-clear">
+                            <mat-icon>clear</mat-icon>
+                        </button>
+                        <button mat-icon-button
+                                [attr.title]="'CORE.METADATA.ACTIONS.SAVE' | translate"
+                                (click)="onSaveTagsChanges($event)"
+                                color="primary"
+                                data-automation-id="save-tags-metadata"
+                                [disabled]="!hasMetadataChanged">
+                            <mat-icon>check</mat-icon>
+                        </button>
+                    </div>
+                    </adf-content-metadata-header>
+            </mat-expansion-panel-header>
+            <div *ngIf="!isEditingTags" class="adf-metadata-properties-tags">
+                <span *ngFor="let tag of tags" class="adf-metadata-properties-tag">{{ tag }}</span>
+            </div>
+            <div *ngIf="showEmptyTagMessage()" class="adf-metadata-no-item-added">
```

Comment:
We raised some time ago jira to avoid calling functions in html template except for events. Can we replace that function's usage with some variables?

Reply:
@DenysVuika not addressed, similarly like comment below. Should we leave that for now/change as part of mentioned jira?

Reply:
@DenysVuika what with this one, should we address that?

Reply:
Done